Add pages for URLPattern API (#8734)
mdn/content: files/en-us/web/api/urlpattern/test/index.md

---
title: URLPattern.test()
slug: Web/API/URLPattern/test
tags:
  - API
  - Method
  - Reference
  - URLPattern
  - URLPattern API
  - Polyfill
  - Experimental
browser-compat: api.URLPattern.test
---

{{APIRef("URLPattern API")}}

The **`test()`**method of the {{domxref("URLPattern")}} interface takes a URL or
object of URL parts, and returns a boolean indicating if the given input matches
the current pattern.

{{AvailableInWorkers}}

## Syntax

```js
test(input);
test(input, baseURL);
```

### Parameters

- `input`

>   - : The URL or URL parts that the matcher will match against. This can either

Reviewer: ```suggestion
  - : The URL or URL parts to match against. This can either
```
